core/src/translate/expr.rs

```diff
@@ -226,7 +255,12 @@ pub fn translate_order_by_expr(sql_order_by_expr: &SqlOrderByExpr) -> Result<Ord
     }
 
     Ok(OrderByExpr {
-        expr: translate_expr(expr)?,
+        expr: translate_expr_with_params(expr, params)?,
         asc: *asc,
     })
 }
+
+pub fn translate_order_by_expr(sql_order_by_expr: &SqlOrderByExpr) -> Result<OrderByExpr> {
```

Comment:
it would be nice to add a simple docs which function returns `Result` For example:
```rust
/// Translates a [`SqlOrderByExpr`] into `GlueSQL's` [`OrderByExpr`] without parameters.
///
/// # Errors
///
/// Returns an error when the order-by expression uses unsupported syntax (for example,
/// NULLS FIRST/LAST) or otherwise cannot be converted.
```

Reply:
Applied!